x-pack/plugins/apm/public/components/app/transaction_details/waterfall_with_summary/waterfall_container/Waterfall/waterfall_item.tsx

```diff
@@ -147,7 +147,12 @@ function HttpStatusCode({ item }: { item: IWaterfallSpanOrTransaction }) {
 function NameLabel({ item }: { item: IWaterfallSpanOrTransaction }) {
   switch (item.docType) {
     case 'span':
-      return <EuiText size="s">{item.doc.span.name}</EuiText>;
+      var name = item.doc.span.name;
+      if(item.doc.span.composite !== undefined){
```

Comment:
```suggestion
      if(item.doc.span.composite){
```

Reply:
This looks like it's going to fail Prettier. `yarn lint:es --fix` should fix it or you can run a prettier extension in your editor.